docker/production/docker-compose.yml

```diff
@@ -10,8 +10,8 @@ services:
     logging:
       driver: "json-file"
       options:
-        max-file: "10"
-        max-size: "200k"
+        max-file: "100"
+        max-size: "2000k"
```

Comment:
is there a reason why we need to increase the number of the log files? (they will get rotated anyway)
max-size: "2m" is more readable
also 
compress: "true" 
is an option

Reply:
I’m fine with this I just shipped something small as to not fill up people’s machines with logs.